policies/coding-style.md

```diff
@@ -323,7 +323,16 @@ because it is a simple way to add valuable information for the reader.
 The name in the prototype declaration should match the name in the function
 definition.
 
-### Chapter 6.1: Extending existing functions
+### Chapter 6.1: Checking function aguments
+
+A _public_ function should verify that it's arguments are sensible.
```

Comment:
(minor) "it's" is short for "it is". The possessive is "its" (like "his", not like "John's")

Reply:
Argh!  I make this typo a lot :(

Reply:
Yes, while I can't not notice the issue when I'm reading text, I find I frequently type both "its" and "it's" incorrectly!